helmfile/upstream/metrics-server/values.yaml

```diff
@@ -108,6 +118,18 @@ service:
   #  kubernetes.io/cluster-service: "true"
   #  kubernetes.io/name: "Metrics-server"
 
+metrics:
+  enabled: false
+
+serviceMonitor:
+  enabled: false
```

Comment:
Would we like to enable these?

Reply:
Hmm, not sure! What about u others?

metrics.enabled = If true, allow unauthenticated access to /metrics.
serviceMonitor.enabled = If true, create a Prometheus service monitor. This needs metrics.enabled to be true.

Reply:
I'm not sure how prometheus scrapes these today? If it's a separate service monitor we install somewhere else then maybe this could replace that one

Reply:
Progression:
We are at abit of standstil and confused about how the metrics server have been providing us metrics in the first place? Have we missed something?